### PR TITLE
SECENG-6135 Add in 403 response for HTTP API docs about WAF

### PR DESCRIPTION
**Pull request by hac-phan-ampl**

### CODE REVIEW

File: content/collections/api/en/http-v2.md [hunk 1]
```diff
@@ -2029,6 +2029,29 @@ Possible reasons for an invalid request:
 | `throttled_devices`                | Object. A map from device_id to its current events per second rate, for all devices that exceed the app's current threshold. |
 | `throttled_events`                 | [integer]. Array of indexes in the events array indicating events whose `user_id` or `device_id` got throttled               |
 
+### 403 (forbidden)
+[403 Forbidden](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3). The request is blocked by our Web Application Firewall (WAF).
```
Comment (markzegarelli) [inline content/collections/api/en/http-v2.md:2033]:
```suggestion

[403 Forbidden](https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3). The request is blocked by Amplitude's Web Application Firewall (WAF).
```

File: content/collections/api/en/http-v2.md [hunk 2]
```diff
@@ -2029,6 +2029,29 @@ Possible reasons for an invalid request:
...
+
+Possible reasons for the response:
+
+- The request contains invalid values within the Header, Body, and/or URI that match our security filters
+- The request is from a sanctioned region that we are not allowed to accept requests from
```
Comment (markzegarelli) [inline content/collections/api/en/http-v2.md:2038]:
```suggestion
- The request is from a sanctioned region from which Amplitude isn't allowed to accept requests.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by hac-phan-ampl** [comment]
@markzegarelli thanks! I guess that's why they don't usually let me write documentation :)

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
All good @hac-phan-ampl! Thanks for the update!

**Pull request merged**